From 5d7c78d3b609744bf62b3b2d3b53d461d1d3afe4 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Thu, 27 Dec 2018 14:58:45 +0800
Subject: [PATCH 38/47] Enable Binder Proxy Tracking by Uid only on eng builds

 * App would be killed once limit reached, this is really bad for users.

Change-Id: I226a7a34031c9e9d8bf6843c22d79e72203bf867
---
 packages/SystemUI/src/com/android/systemui/SystemUIService.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/SystemUIService.java b/packages/SystemUI/src/com/android/systemui/SystemUIService.java
index bb82a54c12f..9c17d78aff2 100644
--- a/packages/SystemUI/src/com/android/systemui/SystemUIService.java
+++ b/packages/SystemUI/src/com/android/systemui/SystemUIService.java
@@ -43,7 +43,7 @@ public class SystemUIService extends Service {
             throw new RuntimeException();
         }
 
-        if (Build.IS_DEBUGGABLE) {
+        if (Build.IS_ENG) {
             // b/71353150 - looking for leaked binder proxies
             BinderInternal.nSetBinderProxyCountEnabled(true);
             BinderInternal.nSetBinderProxyCountWatermarks(1000,900);
-- 
2.17.1

